ac/ac-quiz/src/index.js

```diff
@@ -31,12 +31,38 @@ const exportData = (configData, { payload }) => {
   return compact([headers, ...csv.sort()]).join('\n');
 };
 
+const formatProduct = (configData, itemWrapper) => {
+  const item = itemWrapper['form'];
+  if (item) {
+    const questions = configData.questions.map(q => q.question);
+    const answers = [];
+    const correctQs = [];
+    Object.keys(item).forEach(q => {
+      const num = getNum(q);
+      const response = configData.questions[num].answers[item[q]];
+      answers[num] = response.choice;
+      if (configData.hasAnswers) {
+        correctQs[num] = !!response.isCorrect;
+      }
+    });
+    const correctCount = compact(correctQs).length;
+    const maxCorrect = configData.questions.length;
+    if (configData.hasAnswers) {
+      return { questions, answers, correctQs, correctCount, maxCorrect };
+    } else {
+      return { questions, answers };
+    }
+  }
+};
+
 export default ({
   id: 'ac-quiz',
   type: 'react-component',
   meta,
   config,
+  configUI,
   ActivityRunner,
   dashboard,
-  exportData
+  exportData,
+  formatProduct
```

Comment:
I don't really like the name you chose, and I think the naming is important here as it concerns the API. Did you have other ideas for names? What do you think of `makeProduct` ?

Reply:
Overall I think it would be beneficial to spend some time renaming a few things around, starting maybe from what confused Yannis the most.

Reply:
I agree - I have thought for a while that we should rethink a bit the entire ActivityPackage API, since we keep adding new stuff to it. I'll merge this for now, but let's find some time to think about this.